package.json

```diff
@@ -1,14 +1,31 @@
 {
-  "name": "safe-event-emitter",
+  "name": "@metamask/safe-event-emitter",
   "version": "1.0.1",
-  "description": "",
+  "description": "An EventEmitter that isolates the emitter from errors in handlers",
   "main": "index.js",
+  "publishConfig": {
+    "registry": "https://registry.npmjs.org/",
+    "access": "public"
+  },
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "build": "tsc --project .",
+    "test": "jest",
+    "lint": "eslint . --ext .js,.jsx,.ts,.tsx"
   },
   "author": "",
   "license": "ISC",
+  "devDependencies": {
+    "@metamask/eslint-config": "^1.0.0",
+    "@types/jest": "^24.9.0",
+    "@typescript-eslint/eslint-plugin": "^2.16.0",
+    "@typescript-eslint/parser": "^2.16.0",
+    "eslint": "^6.8.0",
+    "eslint-plugin-jest": "^23.6.0",
+    "jest": "^24.9.0",
+    "ts-jest": "^24.3.0",
+    "typescript": "^3.7.5"
+  },
   "dependencies": {
-    "events": "^3.0.0"
+    "@types/node": "^10.17.13"
```

Comment:
Does this need to be a production dependency? :thinking:

Reply:
Good catch, thanks!